src/events/provider.py

```diff
@@ -106,7 +111,7 @@ def _on_database_requested(self, event: DatabaseRequestedEvent) -> None:
                     event.relation.id, self.context.unit_server.ca_cert
                 )
 
-        except (Exception) as e:
+        except Exception as e:
             logger.exception(e)
             self.charm.unit.status = BlockedStatus(str(e))
```

Comment:
This status seems to be overwritten by other status changes.

Reply:
That's true, because our status are handled centrally. Added to our list of housekeeping tasks. Thanks!